docs/ui/workspaces.md

```diff
@@ -97,4 +97,36 @@ Select **Send** to initiate the invitation.
 
 To add a service account to a workspace, select the Service Accounts **+** icon. You can select from a list of existing service accounts configured for the organization. Select a Workspace Role for the service account. This will be the initial role for the service account within the workspace. A workspace Owner can change this role at any time. Select **Share** to finalize adding the service account.
 
-To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
\ No newline at end of file
+To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
+
+## Workspace transfer
+
+Workspace transfer enables you to move an existing workspace from one account to another. For example, you may transfer a workspace from a personal account to an organization.
+
+Transferring a workspace enables you to take existing configuration and flow run history with you as projects move to different teams or organizations. Without transfer, users moving to a new organization from, for example, a personal account would need to re-create all of their block, deployment, notification, and work queue configuration in a new workspace. Additionally, any flow run history would remain in the personal workspace. 
```

Comment:
I'm curious if it's valuable to spell out what would happen if transfer didn't exist here? In my mind, this framing has been useful internally to motivate why we want to build this functionality, but for the user the functionality already exists, so this is ultimately not relevant to them. (Except potentially in the case someone is weighing creating a new workspace vs. a transfer, but that to me feels niche for someone reading the transfer docs)

Reply:
I think it's useful to be explicit about value and consequences of transfer. Maybe worth rewording.